hyp3_sdk/hyp3.py

```diff
@@ -219,47 +219,120 @@ def prepare_autorift_job(cls, granule1: str, granule2: str, name: Optional[str]
+                                        include_scattering_area=include_scattering_area,
+                                        radiometry=radiometry,
+                                        resolution=resolution,
+                                        scale=scale,
+                                        speckle_filter=speckle_filter,
+                                        **kwargs)
         return self.submit_prepared_jobs(prepared_jobs=job_dict)
 
     @classmethod
-    def prepare_rtc_job(cls, granule: str, name: Optional[str] = None, **kwargs) -> dict:
+    def prepare_rtc_job(cls,
+                        granule: str,
+                        name: Optional[str] = None,
+                        dem_matching: Optional[bool] = None,
+                        include_dem: Optional[bool] = None,
+                        include_inc_map: Optional[bool] = None,
+                        include_scattering_area: Optional[bool] = None,
+                        radiometry: Union[Literal['gamma0'], Literal['sigma0']] = None,
+                        resolution: Literal[30] = None,
+                        scale: Union[Literal['power'], Literal['amplitude']] = None,
+                        speckle_filter: Optional[bool] = None,
+                        **kwargs) -> dict:
         """Submit an RTC job
 
         Args:
             granule: The granule (scene) to use
             name: A name for the job
+            dem_matching: Coregisters SAR data to the DEM, rather than using dead reckoning based on orbit files
+            include_dem: Include the DEM file in the product package
+            include_inc_map: Include the incidence angle map in the product package
+            include_scattering_area: Include the scattering area in the product package
+            radiometry: Backscatter coefficient normalization, either by ground area (sigma0) or illuminated area
+            projected into the look direction (gamma0)
+            resolution: Desired output pixel spacing in meters
+            scale: Scale of output image; either power or amplitude
+            speckle_filter: Apply an Enhanced Lee speckle filter
             **kwargs: Extra job parameters specifying custom processing options
 
         Returns:
             A dictionary containing the prepared RTC job
         """
+        job_parameters = {
+            'dem_matching': dem_matching,
+            'include_dem': include_dem,
+            'include-inc_map': include_inc_map,
+            'include_scatteering_area': include_scattering_area,
+            'radiometry': radiometry,
+            'resolution': resolution,
+            'scale': scale,
+            'speckle_filter': speckle_filter,
+            **kwargs
+        }
+
         job_dict = {
-            'job_parameters': {'granules': [granule], **kwargs},
+            'job_parameters': {'granules': [granule], **{k: v for k, v in job_parameters.items() if v is not None}},
```

Comment:
Maybe instead of `None` as the default, we should actually set the API default and we'd avoid this. We already have to update the SDK anytime we change the option...